pandas/tests/test_nanops.py

```diff
@@ -270,6 +270,7 @@ def _badobj_wrap(self, value, func, allow_complex=True, **kwargs):
                 value = value.astype("f8")
         return func(value, **kwargs)
 
+    @pytest.mark.xfail(reason="GH12863: numpy result won't match for object type")
```

Comment:
do we need this ? (same reason above)

Reply:
Similar response - think more for coverage than matching numpy. Since the nanop gets indirectly tested for all other any/all reductions, I think it could be removed